curriculum/challenges/english/blocks/daily-coding-challenges-javascript/68f6587287ad1f4ad39b0c7d.md

```diff
@@ -0,0 +1,112 @@
+---
+id: 68f6587287ad1f4ad39b0c7d
+title: "Challenge 91: Word Search"
+challengeType: 28
+dashedName: challenge-91
+---
+
+# --description--
+
+Given a matrix (an array of arrays) of single letters and a word to find, return the start and end indices of the word in the matrix.
+
+- The given matrix will be filled with all lowercase letters (`a-z`).
+- The word to find will always be in the matrix exactly once.
+- The word to find will always be in a straight line in one of these directions:
+  - left to right
+  - right to left
+  - top to bottom
+  - bottom to top
+
+For example, given the matrix:
+
+```json
```

Comment:
Should this be `md` instead? Since screen readers announce the code block language, labeling it as JSON could be misleading.

```suggestion
```md
```